gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java

```diff
@@ -18,13 +18,15 @@
  */
 package org.apache.iceberg.gcp;
 
+import com.google.api.client.util.Preconditions;
 import java.io.Serializable;
 import java.util.Date;
 import java.util.Map;
 import java.util.Optional;
 import org.apache.iceberg.util.PropertyUtil;
 
 public class GCPProperties implements Serializable {
+
```

Comment:
unnecessary change

Reply:
@snazy can you please remove that newline change?

Reply:
Nope - it's spottless that re-adds it everytime

Reply:
that's weird, it doesn't do that for me. Also the build would have failed if that code here wouldn't be spotless-compliant

Reply:
PR is ready to go once this diff is reverted